sbd.spec

```diff
@@ -97,6 +105,20 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun sbd.service
 %systemd_postun sbd_remote.service
+
+%else
+
+%post
+/sbin/chkconfig --add sbd_helper || :
```

Comment:
Why do we add sbd_helper but not sbd_remote_helper? Is this even necessary, since we don't use Default-{Start,Stop}?